pymc/distributions/distribution.py

```diff
@@ -353,17 +352,11 @@ def dist(
         # Create the RV with a `size` right away.
         # This is not necessarily the final result.
         rv_out = cls.rv_op(*dist_params, size=create_size, **kwargs)
-        rv_out = maybe_resize(
-            rv_out,
-            cls.rv_op,
-            dist_params,
-            ndim_expected,
-            ndim_batch,
-            ndim_supp,
-            shape,
-            size,
-            **kwargs,
-        )
+
+        # Replicate dimensions may be prepended via a shape with Ellipsis as the last element:
+        if shape is not None and Ellipsis in shape:
```

Comment:
do we check anywhere that the ellipsis is in the last position?

Reply:
Yes, there are functions further above that do this.
Also that's the difference between `WeakShape` and `StrongShape` so mypy also tells us about errors.
Actually I ran mypy before pushing and it was satisfied.

If only I had run pre-commit the right way. It's still messing with the environment YAML files every time - we have some kind of line ending problem in the hook. So it's always failing for me and sometimes I overlook black problems because of that.